code/modules/power/engines/supermatter/supermatter_event.dm

```diff
@@ -137,14 +137,14 @@
 	name = "B-1"
 
 /datum/supermatter_event/bravo_tier/gas_multiply/on_start()
-	supermatter.gas_multiplier = 1.5
+	supermatter.gas_multiplier = 3
 
 
 /datum/supermatter_event/bravo_tier/heat_multiplier
 	name = "B-2"
 
 /datum/supermatter_event/bravo_tier/heat_multiplier/on_start()
-	supermatter.heat_multiplier = 1.25
+	supermatter.heat_multiplier = 3
```

Comment:
This one is not double 😃 

![image](https://github.com/user-attachments/assets/44a71a1d-ad1d-4206-be72-c2f2eb526b20)

Reply:
good nuff for government work